HOW-TO-DEV.md

```diff
@@ -0,0 +1,38 @@
+# Setup
+Download the latest version of IntelliJ IDEA (https://www.jetbrains.com/idea/download/)
+Configure to the correct version of Node.js (https://nodejs.org/en/download/)
+Configure to the correct version of Java (https://www.oracle.com/java/technologies/javase-downloads.html)
```

Comment:
I would link the Mac compatibile build here instead. https://adoptium.net/temurin/releases/
Oracle BAD.